pkg/security/filters/required_authn.go

```diff
@@ -58,14 +63,10 @@ func (raf *requiredAuthnFilter) FilterMatchers() []web.FilterMatcher {
 					web.ServicePlansURL+"/**",
 					web.VisibilitiesURL+"/**",
 					web.NotificationsURL+"/**",
+					web.ConfigURL+"/**",
```

Comment:
You can remove `web.LoggingConfigURL+"/**"` below as this declaration already covers any subpaths